build.override.targets

```diff
@@ -3,37 +3,50 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
   </PropertyGroup>
-  
+
+  <PropertyGroup>
+    <UseLatestDependencies Condition="'$(UseLatestDependencies)'==''">false</UseLatestDependencies>
+  </PropertyGroup>
+
+  <PropertyGroup Condition = "'$(UseLatestDependencies)' == 'true'" >
+    <ScenarioDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\latest.dependencies.json</ScenarioDependencyJson>
+  </PropertyGroup>
+
+  <!-- If $(UseLatestDependencies) is 'false' and $(ScenarioDependencyJson) has not been set in command line use 'stable.dependencies.json' as the default value. -->
+  <PropertyGroup Condition = "'$(ScenarioDependencyJson)' == ''" >
```

Comment:
This property is supposed to be the file name of the json file, so I wouldn't add a 'false' test

---
In reply to: [72516176](https://github.com/dotnet/wcf/pull/1399#discussion_r72516176) [](ancestors = 72516176)